include/bluetooth/mesh/model_types.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+/**
+ * @file
+ * @defgroup bt_mesh_model_types Common model types
+ * @{
+ * @brief Collection of types exposed in the model definitions.
+ */
+
+#ifndef BT_MESH_MODEL_TYPES_H__
+#define BT_MESH_MODEL_TYPES_H__
+
+#include <bluetooth/mesh.h>
+
+/** Maximum permissible transition time in milliseconds */
+#define BT_MESH_MODEL_TRANSITION_TIME_MAX_MS (K_MINUTES(10) * (0x3e))
+
+/** Generic Transition parameters for the model messages. */
+struct model_transition {
+	u32_t time; /**< Transition time value in milliseconds */
+	u32_t delay; /**< Message execution delay in milliseconds */
+};
+
+/**
+ * Transaction ID structure, storing information about the previous
+ * transaction in model spec messages.
+ */
+struct transaction_id {
```

Comment:
Is "Transaction ID" a mesh definition? Calling it ID seems strange, also since it contains a tid, looks more like a transaction_ctx.

Reply:
Agreed